tools/eslint/src/check-license.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2024 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {ESLintUtils} from '@typescript-eslint/utils';
+
+const createRule = ESLintUtils.RuleCreator(name => {
+  return `https://github.com/puppeteer/puppeteer/tree/main/tools/eslint/${name}.js`;
```

Comment:
I think other places are wrong.
```suggestion
  return `https://github.com/puppeteer/puppeteer/tree/main/tools/eslint/${name}.ts`;
```